criu/action-scripts.c

```diff
@@ -31,6 +31,7 @@ static const char *action_names[ACT_MAX] = {
 	[ACT_POST_RESUME] = "post-resume",
 	[ACT_ORPHAN_PTS_MASTER] = "orphan-pts-master",
 	[ACT_STATUS_READY] = "status-ready",
+	[ACT_QUERY_EXT_FILES] = "query-ext-files",
```

Comment:
pls add a description of this new hook in Documentation/criu.txt.

Reply:
Done.